docs/dataformat.rst

```diff
@@ -230,6 +234,41 @@ for these extractors. Currently, all intent classifiers make use of available re
     training data!
 
 
+Lookup Tables
+-------------
+Lookup tables in the form of external files can also be specified in the training data.  The externally supplied lookup tables must be in a newline-separated format.  For example, ``data/test/lookup_tables/plates.txt`` may contain
+
+    tacos
+    beef
+    mapo tofu
+    burrito
+    lettuce wrap
+    ...
```

Comment:
you can do an `.. include` here (can't remember the exact format, there's an example somewhere in the docs), so it just includes the whole text file

Reply:
fixed